.travis.yml

```diff
@@ -7,6 +7,6 @@ script:
 
 jobs:
   include:
-    - stage: pre-commit tests
-      script:
-      - pre-commit run --all-files
+  - stage: pre-commit tests
+    script:
+    - pre-commit run --all-files
```

Comment:
@webknjaz What is it? I don't hear about it. 
I have one more question about JWT. I create token and return it. But what goes on with him? The front-end stores this token in the user's browser, or what? I don't understand this moment.

Reply:
1) it's an uncompromising code formatter, Google it
2) yes, front-end verifies the token and payload, then it saves it for future use and finally it sends it along with all requests to the backed API. It can use either HTTP header or any other convenient way you agree to use.